ninja-core/src/main/java/ninja/migrations/flyway/MigrationEngineFlyway.java

```diff
@@ -41,12 +41,19 @@ public void migrate() {
         String connectionUrl = ninjaProperties.getOrDie(NinjaConstant.DB_CONNECTION_URL);
         String connectionUsername = ninjaProperties.getOrDie(NinjaConstant.DB_CONNECTION_USERNAME);
         String connectionPassword = ninjaProperties.getOrDie(NinjaConstant.DB_CONNECTION_PASSWORD);
+        String locations = ninjaProperties.get(NinjaConstant.NINJA_MIGRATION_LOCATIONS);
+        String schemas = this.ninjaProperties.get(NinjaConstant.NINJA_MIGRATION_SCHEMAS);
 
         // We migrate automatically => if you do not want that (eg in production)
         // set ninja.migration.run=false in application.conf
         Flyway flyway = new Flyway();
         flyway.setDataSource(connectionUrl, connectionUsername, connectionPassword);
 
+        if (locations != null)
+            flyway.setLocations(locations);
+        if (schemas != null)
```

Comment:
could you think of a good way to test it? if it is too complicated I think we are ok with no test...

Reply:
I think the easiest way is to use a mocked Flyway instance and test the method invocations. Or maybe the best way is to do full integration tests (with h2 database), but it's more complicated. What do you think?

Reply:
I added some tests. Please take look at https://github.com/ninjaframework/ninja/pull/668/commits/27f4df3c9d2b2dab73f893391201c9e2657bab42